src/main/core-options.js

```diff
@@ -62,6 +62,34 @@ const options = {
     `,
     cliCategory: CATEGORY_EDITOR
   },
+  eol: {
+    since: "1.15.0",
+    category: CATEGORY_GLOBAL,
+    type: "choice",
+    default: "auto",
+    description: "Which end of line characters to apply.",
+    exception: value => typeof value === "string",
+    choices: [
+      {
+        value: "auto",
+        description: dedent`
+          Maintain existing
+          (mixed values within one file are normalised by looking at what's used after the first line)
+        `
+      },
+      { value: "cr", description: "Carriage Return character only (\\r)" },
```

Comment:
I agree that this choice is ancient, so probably not including it will not harm anyone. However, `cr` is one of accepted values for `end_of_line` in [`.editorconfig`](https://editorconfig.org/), so removing it will break consistency. If you think we should get rid of `cr` despite that, what shall we do if someone uses it in `.editorconfig`?

Reply:
Given that you've marked this conversation as resolved, I put a checkbox next _Decide if we want to support `--endOfLine=cr` at all_ in the first comment.